vignettes/ebola.Rmd

```diff
@@ -0,0 +1,615 @@
+```
+
+To prepare the data for use with the latent individual model, we use the function `as_latent_individual`:
+
+```{r}
+obs_prep <- as_latent_individual(obs_cens)
+```
+
+Now we are ready to fit the latent individual model.
+
+### Intercept only model
+
+We set `formula` to `brms::bf(mu ~ 1, sigma ~ 1)` to place an intercept only model on the `mu` and `sigma` parameters of the lognormal distribution.
+This distribution is specified using `family = brms::lognormal()`.
+We use the Laplace approximation via `algorithm = "laplace"` (see the approximate inference vignette) for fast model fitting.
+
+```{r}
+fit <- epidist(
+  data = obs_prep,
+  formula = brms::bf(mu ~ 1, sigma ~ 1),
+  family = brms::lognormal(),
+  algorithm = "laplace",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+```
+
+The `fit` object is a [`brmsfit`](https://paulbuerkner.com/brms/reference/brmsfit-class.html) object, and has the associated range of methods.
+See `methods(class = "brmsfit")` for more details.
+For example, we may use `summary` to view information about the fitted model, including posterior estimates for the regression coefficients:
+
+```{r}
+summary(fit)
+```
+
+### Sex-stratified model
+
+To fit a model which varies `mu` and `sigma` by sex we alter the `formula` specification to include fixed effects for sex as follows:
+
+```{r}
+fit_sex <- epidist(
+  data = obs_prep,
+  formula = brms::bf(mu ~ 1 + sex, sigma ~ 1 + sex),
+  family = brms::lognormal(),
+  algorithm = "laplace",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+```
+
+A summary of the model shows that males tend to have longer delays (the posterior mean of `sexM` is `r signif(summary(fit_sex)$fixed[3, "Estimate"], 3)`) and greater delay variation (the posterior mean of `sigma_sexM` is `r signif(summary(fit_sex)$fixed[4, "Estimate"], 3)`):
+
+```{r}
+summary(fit_sex)
+```
+
+### Sex-district stratified model
+
+Finally, we will fit a model which also varies by district.
+To do this, we will use district level random effects, assumed to be drawn from a shared normal distribution, within the model for both the `mu` and `sigma` parameters:
+
+```{r}
+fit_sex_district <- epidist(
+  data = obs_prep,
+  formula = brms::bf(
+    mu ~ 1 + sex + (1 | district),
+    sigma ~ 1 + sex + (1 | district)
+  ),
+  family = brms::lognormal(),
+  algorithm = "laplace",
+  refresh = 0,
+  silent = 2,
+  seed = 1
+)
+```
+
+Along with looking at the `summary`, we may also use the `brms::ranef` function to look at the estimates of the random effects:
+
+```{r}
+summary(fit_sex_district)
+brms::ranef(fit_sex_district)
+```
+
+## Posterior expectations
+
+To go beyond simple summaries of the fitted model, we recommend using the `tidybayes` package.
+For example, to obtain the posterior expectation of the delay distribution, under no censoring or truncation, we may use the `modelr::data_grid` function in combination with the `tidybayes::add_epred_draws` function.
```

Comment:
if you use `modelr::data_grid()` and so on it will auto link

Reply:
[modelr::data_grid()] or `modelr::data_grid()`?

Reply:
I think the latter

Reply:
https://github.com/epinowcast/epidist/pull/52/commits/0a76c6daa43216b162f1bc2fe40e6f1541952ad1